consensus/src/main/java/org/ethereum/beacon/consensus/spec/BlockProcessing.java

```diff
@@ -30,7 +30,8 @@ default void process_block_header(MutableBeaconState state, BeaconBlock block) {
     // Verify that the slots match
     assertTrue(block.getSlot().equals(state.getSlot()));
     // Verify that the parent matches
-    assertTrue(block.getPreviousBlockRoot().equals(signed_root(state.getLatestBlockHeader())));
+    // FIXME: signed_root should match
```

Comment:
Let's not forget this thing

Reply:
I'd better create an issue and point this out in the issue.